src/sql/base/browser/ui/scrollableSplitview/scrollableSplitview.ts

```diff
@@ -611,9 +611,9 @@ export class ScrollableSplitView extends HeightMap implements IDisposable {
 		} else {
 			item.size = size;
 			this.updateSize(item.view.id!, size);
-			let top = item.top + item.size;
-			for (let i = index + 1; i < this.viewItems.length; i++) {
-				let currentItem = this.viewItems[i];
+			let top: number = 0;
```

Comment:
This is removing the optimization in not moving every item when you don't need to. Not sure this is actually a bug.

Reply:
this is try to resolve a bug where the `top` is incorrect in the heightmap.  The code that updates items in the heightmap isn't correctly maintaining the `top` value which is causing issue with rendering when there are multiple resultssets.  You can run a query with several SQL statements and see how grids are rendered when scrolling to repo.